### ISSUE TITLE
TROY does not use the 'automatic sandbox' of sagapilot for the home directory

### ISSUE THREAD

**Issue by mturilli** [radical-cybertools/radical.owms#51]
On both FutureGrid and XSEDE, if the home directory on the remote resource is not defined either as 'home' in .troy.cfg or as task_descr.working_directory (branch master).

**Comment by andre-merzky** [comment]
I am not sure I understand that one: if `task_description.working_directory` is specified for a task, then it should not use the sandbox, right?  I'll check on the `home` -- specifying `home` should not trigger the sandbox, we should make that option explicit (`pilot_working_directory`).

**Comment by mturilli** [comment]
Apologies, my description of the problem was not clear. Let me try again
but please let me know whether it still is confused. If I do not specify a
'home' in the configuration file and if I do not specify a
task_description.working_directory in my script, I would expect TROY to let
sagapilot to decide what is my home directory.

On Mon, Feb 17, 2014 at 2:21 AM, Andre Merzky notifications@github.comwrote:

> I am not sure I understand that one: if task_description.working_directoryis specified for a task, then it should not use the sandbox, right? I'll
> check on the home -- specifying home should not trigger the sandbox, we
> should make that option explicit (pilot_working_directory).
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/saga-project/troy/issues/51#issuecomment-35233592
> .

## 

Dr Matteo Turilli
Department of Electrical and Computer Engineering
Rutgers University

**Comment by andre-merzky** [comment]
Indeed, I would expect the same :)  

Thanks!  A.
